addons/web/static/src/js/views/control_panel/search/search_filters.js

```diff
@@ -206,6 +206,9 @@ var DateTime = Field.extend({
      * @return {Moment} The value in UTC
      */
     get_value: function (index) {
+        if (!this["datewidget_" + (index || 0)].getValue()) {
```

Comment:
[FIX] web: custom search on date/datetime field without value
Before this commit: if a user would not enter a value when creating a custom filter of type date/datetime, he would get a traceback.
After this commit: the traceback no longer appear in the same situation and the absence of value is interpreted as false.

task-2340986